visualizer/visualizer.py

```diff
@@ -14,6 +14,7 @@
 import json
 import yaml
 import numpy
+from create_DB import *
```

Comment:
do we need to install python module for this? if so, please also update at readme.md